src/utils.ts

```diff
@@ -280,15 +280,15 @@ export class Utils {
 
   /** @internal */
   static getScrollParent(el: HTMLElement): HTMLElement {
-    let returnEl;
-    if (el === null) {
-      returnEl = null;
-    } else if (el.scrollHeight > el.clientHeight) {
-      returnEl = el;
+    if (el === null) return document.documentElement;
+    const style = getComputedStyle(el);

```

Comment:
Yes, also with `el.scrollHeight > el.clientHeight` sometimes is selecting a non scrollable parent.

Reply:
interesting. didn't think that would be possible... but I realize the call you have return the first that could scroll which is safer anyway... but always return the doc as least, which might not scroll so it should return NULL and have that be handled ideally...

Reply:
Exactly, the best option would be returning `NULL` when there isn't a scrollable parent but in this case, we need to prevent `gridstack` to continue growing.

I was using the example `float.html` to check my changes. I think in this example the ideal would be that `gridstack` stops growing when there is no more page because the element that is scrolling is the document not the container that wraps `gridstack`.

Reply:
And we need to manually prevent `gridstack` to grow because:
In my case, I have two screens one on top of another and I have the browser with the webpage with gridstack on the screen above.
![Captura de pantalla 2021-01-03 a las 8 24 12](https://user-images.githubusercontent.com/26092748/103473864-6256ec00-4d9d-11eb-9a87-1a079d9e827e.png)

When I try to resize or drag and drop an item to the bottom my mouse continue on the screen bellow so `gridstack´ still growing even when the page can't scroll. Once I stop there is no scrollable parent so the `documentElement` is resized to wrap everything.

Reply:
well we have a `maxRow` to prevent grid from growing past that size, but you are saying that if none of the parents can scroll (by default divs will auto scroll unless you have `overflow: hidden` or something) then we should stop growing so we don't clip, which makes sense... Looks like you are checking for overflow | overflow-y for and auto or scroll
you don't need 2 screens to test this. browser smaller than full screen will do that too...

if you want to take a stab at it that would be great. `_prepareDragDropByNode()` should cache if we can not v-scroll and prevent move/size past the max somehow.

Reply:
I'll take a look at this over the week (probably weekend, I'm quite busy right now) :)